<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditEntryWidgetNotes</class>
 <widget class="QWidget" name="EditEntryWidgetNotes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="notesLabel">
     <property name="text">
      <string>Notes:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="notesEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
